[2024-03-15 19:05:07,510] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-15T19:04:00+00:00 [queued]>
[2024-03-15 19:05:07,538] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-15T19:04:00+00:00 [queued]>
[2024-03-15 19:05:07,538] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 19:05:07,538] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-15 19:05:07,538] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 19:05:07,556] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_dt_score> on 2024-03-15T19:04:00+00:00
[2024-03-15 19:05:07,568] {standard_task_runner.py:52} INFO - Started process 53530 to run task
[2024-03-15 19:05:07,573] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_dt_score', '2024-03-15T19:04:00+00:00', '--job-id', '3612', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp3vekmfmq', '--error-file', '/tmp/tmp93w4u4wt']
[2024-03-15 19:05:07,574] {standard_task_runner.py:77} INFO - Job 3612: Subtask compute_dt_score
[2024-03-15 19:05:07,644] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_dt_score 2024-03-15T19:04:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 19:05:07,725] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_dt_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T19:04:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T19:04:00+00:00
[2024-03-15 19:05:08,035] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 19:05:08,054] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_dt_score, execution_date=20240315T190400, start_date=20240315T190507, end_date=20240315T190508
[2024-03-15 19:05:08,104] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-15 19:05:08,148] {local_task_job.py:151} INFO - Task exited with return code 0
